docs/data/routes/docs/fundamentals/services/app-configuration/en.md

```diff
@@ -71,5 +71,6 @@ The `defaults` app configuration can be found in `\App_Config\Sitecore\JavaScrip
 | `serverSideRenderingEngine`         | [Rendering engine](/docs/fundamentals/services/view-engine) used by Sitecore when rendering the app server-side. Possible values: `nodejs` and `http`. <br/><br/>Default: `nodejs` |
 | `serverSideRenderingFunctionName`   | Only relevant when `nodejs` rendering engine is enabled. Name of the exported JS function on your `serverBundle` to call when rendering in integrated mode. <br/><br/>Default: `renderView` |
 | `serverSideRenderingEngineEndpointUrl` | Only relevant when `http` rendering engine is enabled. Defines an HTTP endpoint for a remote rendering host. <br/><br/>Default: `` |
+| `serverSideRenderingEngineApplicationUrl` | Only relevant when `http` rendering engine is enabled. Defines the URL that will be used when creating absolute links in the rendered HTML. Link replacement is necessary in order for relative links in the rendered HTML to be able to load in the Experience Editor. If not specified (default), link replacement will not be performed.<br/><br/>Default: `` |
```

Comment:
This is incorrect actually, right @OlegJytnik? By default it will extract the host name from `serverSideRenderingEngineEndpointUrl` I believe.

Reply:
I pulled this direct from the Sitecore config comment :) But yeah, now that you mention it, a separate `EnableRelativeLinkProcessing` config was added later to control this behavior. I'll let Oleg confirm.

Reply:
> This is incorrect actually, right @OlegJytnik? By default it will extract the host name from `serverSideRenderingEngineEndpointUrl` I believe.

Correct. It is default behavior.

Reply:
Fixed.